settings.py

```diff
@@ -151,7 +151,8 @@
 SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
 
 # time in minutes before forced logout due to inactivity
-INACTIVITY_TIMEOUT = 30
+INACTIVITY_TIMEOUT = 60 * 24 * 14
```

Comment:
seems like having it in settings is fine, no? i don't think we'd ever need to make this a per-domain config value